xtask/codegen/src/language_kind.rs

```diff
@@ -0,0 +1,175 @@
+use crate::css_kinds_src::CSS_KINDS_SRC;
+use crate::grit_kinds_src::GRIT_KINDS_SRC;
+use crate::html_kinds_src::HTML_KINDS_SRC;
+use crate::js_kinds_src::JS_KINDS_SRC;
+use crate::json_kinds_src::JSON_KINDS_SRC;
+use crate::kind_src::KindsSrc;
+use proc_macro2::{Ident, Span, TokenStream};
+use quote::quote;
+use std::str::FromStr;
+
+pub const LANGUAGE_PREFIXES: [&str; 7] = ["js_", "ts_", "jsx_", "tsx_", "css_", "json_", "html_"];
```

Comment:
I think we should add `grit_` too.

```suggestion
pub const LANGUAGE_PREFIXES: [&str; 7] = ["js_", "ts_", "jsx_", "tsx_", "css_", "json_", "grit_", "html_"];
```